src/server/pps/server/api_server.go

```diff
@@ -1951,7 +1951,7 @@ func (a *apiServer) CreatePipelineInTransaction(
 			if err := a.env.PFSServer.CreateRepoInTransaction(txnCtx,
 				&pfs.CreateRepoRequest{
 					Repo:        client.NewRepo(input.Cron.Repo),
-					Description: fmt.Sprintf("Cron tick repo for pipeline %s.", request.Pipeline.Name),
+					Description: fmt.Sprintf("Cron tick repo for pipeline %s/%s.", projectName, pipelineName),
```

Comment:
If we had a utility function, we could just use the pipeline and not have the separate project name variable.